quipucords/api/aggregate_report/model.py

```diff
@@ -37,62 +39,98 @@
 logger = logging.getLogger(__name__)
 
 UNKNOWN: str = "unknown"  # placeholder string for missing names/versions/kinds.
+AGGREGATE_REPORT_SKIP_ATTRS: list = ["id", "report"]
 
 
-@dataclass
-class AggregateReport:
+class AggregateReport(BaseModel):
     """Results of the aggregate report."""
 
+    report = models.OneToOneField(
+        "Report", models.CASCADE, related_name="aggregate_report", null=True
```

Comment:
in which scenario would we expect an aggregate report to exist without a report?

(looking at this `null=True`)

Reply:
good point. 👍

Reply:
If there's no particular reason for setting this relationship with `null=True`, could you set it to `False`?
```suggestion
        "Report", models.CASCADE, related_name="aggregate_report"
```
(I think it is false by default)